packages/ember-routing/lib/location/hash_location.js

```diff
@@ -53,7 +52,9 @@ export default EmberObject.extend({
     @since 1.5.1
     @method getHash
   */
-  getHash: EmberLocation._getHash,
+  getHash() {
```

Comment:
Why not `getHash,` here?

Reply:
`utils/getHash` expects location object, while this `class getHash` does not expect any arguments

Reply:
Gotcha, thank you!